recipes/FileUploadHalogenHooks/README.md

```diff
@@ -0,0 +1,9 @@
+# FileUploadHalogenHooks
+
+A Halogen port of the ["Files - Upload" Elm Example](https://elm-lang.org/examples).
```

Comment:
Thoughts on linking directly to the example?
https://elm-lang.org/examples/upload

Reply:
Initially, I was hesitant on doing that because one can't find the rest of the Elm examples if we do that. But I realized there's a "More examples `here`" in the upper-left corner, so that's not true...

Sure, let's update the Readmes to point directly to the code.